apps/site/lib/site_web/controllers/schedule/line_api.ex

```diff
@@ -21,7 +24,17 @@ defmodule SiteWeb.ScheduleController.LineApi do
         stops_by_route_fn: &StopsRepo.by_route/3
       })
 
-    json(conn, Enum.map(line_data, &update_route_stop_data/1))
+    conn =
+      conn
+      |> assign(:route, LineHelpers.get_route(route_id))
+      |> assign_alerts([])
```

Comment:
I forgot this helper existed!

It looks like we might want to filter by direction here. I don't think direction_id is set in the conn here like it is in the query params. If it's not already set in the conn, then we should do something like you did for `:route` and then the helper should filter by direction.

https://github.com/mbta/dotcom/blob/4014bdd53b6a93d3ab8e3c6c7b8826af3eb568c9/apps/site/lib/site_web/controllers/helpers.ex#L86